### ISSUE TITLE
Clicking "Setup SOCKS5 proxy" with the firewall disabled makes the app think you're using Orbot

### ISSUE THREAD

**Issue by Dyras** [celzero/rethink-app#854]
tldr: Disable the firewall and try to enable SOCKS5 proxy. It thinks you have Orbot enabled.

Long version: 
If I go into the settings and click Proxy, I get this message:
"Cannot setup proxies when VPN is running in DNS only mode or lockdown mode. Go to VPN settings"

Aside from that message being super unclear as to what is wrong and I'm pretty sure it leads you to the wrong place, there's a bug when you try to enable the proxy anyway:
If you click the grayed out "Setup SOCKS5 Proxy" you get this message: "Connected to Orbot. Turn it off to continue"

I've never used Rethink with Orbot, nor have I used Orbot at all for that matter, so the message is definitely a mistake.

PS. It took me 3 minutes of looking to find how to enable the firewall if you're connected to the DNS only. The switch next to "Stop" should probably be changed. Maybe to "DNS only" or something?

**Comment by ignoramous** [comment]
Thanks. I can see why the current UX is confusing.

*VPN lockdown mode* is when *Block connections without VPN* is enabled. Tapping on the underlined warning text should take you to the screen where you can disable that setting. If that's not the case, let us know, we should fix it.

That said, we intend to let users connect to a remote socks5 endpoint as that's possible even when the VPN is in a lockdown.

**Comment by ignoramous** [comment]
>  "Cannot setup proxies when VPN is running in DNS only mode or lockdown mode. Go to VPN settings"

Oh, so you might have setup rdns just in *DNS-only mode*. You can change this by tapping the down arrow next to `START` / `STOP` button on the homescreen.
